src/index.ts

```diff
@@ -1,58 +1,102 @@
-export { default as Surface, Props as SurfaceProps } from './container/Surface';
-export { default as Layer, Props as LayerProps } from './container/Layer';
+// "export type" declarations are being used to workaround babel
+// issue  declarations are being used to workaround babel issues 11464 12578
+//
+// see https://github.com/babel/babel/issues/11464#issuecomment-617606898
```

Comment:
Here is the fix. I tested the build locally and the referenced types are now being erased as intended.